Gemfile

```diff
@@ -42,6 +42,10 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :development do
+  # awesome print for rails console
+  gem 'awesome_print'
```

Comment:
I think you could put this in the `group :development` block right above this one.

Reply:
moved awesome_print above